clients/client/src/main/java/org/projectnessie/client/ClientContentsApi.java

```diff
@@ -18,57 +18,20 @@
 import javax.validation.constraints.NotNull;
 
 import org.projectnessie.api.ContentsApi;
-import org.projectnessie.client.http.HttpClient;
 import org.projectnessie.error.NessieConflictException;
 import org.projectnessie.error.NessieNotFoundException;
+import org.projectnessie.model.CommitMeta;
 import org.projectnessie.model.Contents;
 import org.projectnessie.model.ContentsKey;
-import org.projectnessie.model.MultiGetContentsRequest;
-import org.projectnessie.model.MultiGetContentsResponse;
-
-class ClientContentsApi implements ContentsApi {
-
-  private final HttpClient client;
-
-  public ClientContentsApi(HttpClient client) {
-    this.client = client;
-  }
-
-  @Override
-  public Contents getContents(@NotNull ContentsKey key, String ref) throws NessieNotFoundException {
-    return client.newRequest().path("contents").path(key.toPathString())
-                 .queryParam("ref", ref)
-                 .get()
-                 .readEntity(Contents.class);
-  }
-
-  @Override
-  public MultiGetContentsResponse getMultipleContents(@NotNull String ref, @NotNull MultiGetContentsRequest request)
-      throws NessieNotFoundException {
-    return client.newRequest().path("contents")
-                 .queryParam("ref", ref)
-                 .post(request)
-                 .readEntity(MultiGetContentsResponse.class);
-  }
 
+/**
+ * Extends ContentsApi with some helper methods for clients.
+ */
+public interface ClientContentsApi extends ContentsApi {
```

Comment:
This extends `ContentsApi` to provide helper functions for updating and deleting single contents. Currently only tests use these methods so not sure if its really required to have them or if extending the interface is the cleanest way